libs/common/setup.go

```diff
@@ -49,7 +51,12 @@ func Setup(serviceName, version string, auth bool) {
 	}
 
 	if auth {
-		setupKeycloak()
+		if strings.ToLower(hwutil.GetEnvOr("FAKE_TOKEN_ENABLE", "false")) == "true" && Mode == DevelopmentMode {
```

Comment:
staging will run in production mode, yet it should accept fake tokens, right?

Also maybe prefix the env name with INSECURE_